project/Tests/Snapper.MSTest.Tests/MSTestSnapperTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Runtime.CompilerServices;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Snapper.Attributes;
+
+namespace Snapper.MSTest.Tests
+{
+    [TestClass]
+    public class MSTestSnapperTests
+    {
+        [TestMethod]
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        public void TestIfStoredSnapshotIsMatching()

```

Comment:
As mentioned above please move to the new `Snapper.TestFrameworkSupport.Tests` project. 😄